<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_finish</class>
 <widget class="QDialog" name="Dialog_finish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>作成中</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_finish">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ただいま作成中です、しばらくお待ちください。</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_finish">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>完了</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic_finish">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
